### PR TITLE
Fix the "DNS" text invisible issue while in night mode.

### PR DESCRIPTION
**Pull request by TchaikovDriver**
By default, the color of `TextView` would be changed to white while the app is in night mode, which makes the text invisible in  white background.
As Android provides a default color schemes of views for night mode, simply using the default scheme should work.

- Use default background color of `CardView` in Settings, fixing the text color issue in night mode.
- Add a button in `MainActivity` for displaying proxy port, clicking the button would copy the port to clipboard.

### CODE REVIEW

**Comment by wongsyrone** [comment]
Why it's closed?

**Comment by TchaikovDriver** [comment]
> Why it's closed?

I want to commit a workaround for #349 , to show the port in `MainActivity` and provide a copy button.

**Comment by wongsyrone** [comment]
LGTM.

File: app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java [hunk 1]
```diff
@@ -555,6 +569,17 @@ public void onClick(DialogInterface dialog, int which) {
         });
     }
 
+    @UiThread
+    private void updatePortInfo(long port) {
+        currentProxyPort = port;
+        if (INVALID_PORT == port) {
```
Comment (wongsyrone) [inline app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java:571]: How about limit the port range to [0, 65535]

Comment (TchaikovDriver) [reply app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java:571]: Reasonable

**Pull request merged**